src/Player.js

```diff
@@ -1,9 +1,14 @@
 import io from "socket.io-client";
 
-const SPEED = 5;
+const blockHeight = window.innerHeight / 9; 
+const blockWidth = window.innerWidth / 16;
```

Comment:
duplicated information, but it should be fine for now

Reply:
This was what I needed about 20 hours ago